CMakeLists.txt

```diff
@@ -20,6 +20,16 @@ include(GNUInstallDirs)
 list(APPEND CMAKE_MODULE_PATH /usr/local/share/cmake/Modules)
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
+# Include Ogre cmake file on Arch linux
+execute_process(
+  COMMAND uname --kernel-release
```

Comment:
`uname --kernel-release` on Ubuntu gives the kernel version. Does this output something different on Arch?

Reply:
This is the usual output on arch: 5.0.3-arch1-1-ARCH
By the way, I do not know if there any other method to determine the operating system on arch, @julianoes  Do you know any other method??